src/aiodynamo/expressions.py

```diff
@@ -378,6 +389,14 @@ def __invert__(self) -> Condition:
     def encode(self, params: Parameters) -> str:
         pass
 
+    def debug(self, numeric_type: Callable[[str], Any] = float) -> str:
```

Comment:
How about using this in `__repr__`?

Reply:
I actually quite like the current dataclass-provided `__repr__`. I often find it easier to read when there's lots of &'ing and |'ing happening.